[Cloud] Managing pollers (#2819)
centreon/centreon-documentation: cloud/installation/poller-update.md

---
id: poller-update
title: Updating a poller
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

This procedure describes how to update a poller from one minor version to another (e.g. from 23.10.x to 23.10.y).

## Updating a poller

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

1. Clean the cache:

  ```shell
  dnf clean all --enablerepo=*
  ```

2. Then upgrade all the components with the following command:

  ```shell
  dnf update centreon\* --exclude=centreon-plugin*
  ```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

1. Clean the cache:

  ```shell
  dnf clean all --enablerepo=*
  ```

2. Then upgrade all the components with the following command:

  ```shell

>   dnf update centreon\* --exclude=centreon-plugin*

Reviewer: ```suggestion
  dnf update -y centreon\* --exclude=centreon-plugin*
```

---
id: poller-update
title: Updating a poller
---

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

## Poller version

To know the exact version of your poller, run the following command:

```shell

> rpm -qa |grep centreon

Reviewer: ```suggestion
rpm -qa | grep centreon-poller
```

```

The output will look like this:

```shell

> 

Reviewer: ```suggestion
centreon-poller-23.04.10-1.el8.noarch
```

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> This procedure describes how to update a poller from one minor version to another (e.g. from 23.10.x to 23.10.y).

Reviewer: ```suggestion
This procedure describes how to update a poller from one minor version to another (e.g. from AA.BB.xx to AA.BB.yy).
```
Is it more common to use this kind of syntax?
